packages/playground/src/layouts/RootLayout.tsx

```diff
@@ -181,13 +181,21 @@ export const RootLayout: React.FC<RootLayoutProps> = ({ currentUser }) => {
               </Navbar.AccordionGroup>
             </Navbar.Content>
             <Navbar.Footer>
-              <NavLink to="/sandbox/settings/profile/general" className="p-2 hover:bg-tertiary">
-                <NavbarUser.Root
-                  username={currentUser ? currentUser.display_name : 'Steven M.'}
-                  isAdmin={currentUser ? currentUser.admin : false}
-                  url={currentUser ? undefined : '../images/user-avatar.svg'}
-                />{' '}
-              </NavLink>
+              <NavbarUser.Root
+                username={currentUser ? currentUser.display_name : 'Steven M.'}
```

Comment:
is this RootLayout used in Gitness? if so, we should not have hardcoded fallbacks like `Steven M`. Let's fallback to user email if username is not available.